pineappl_cli/src/evolve.rs

```diff
@@ -70,6 +70,7 @@ mod eko {
         V0(MetadataV0),
         V1(MetadataV1),
         V2(MetadataV2),
+        V3(MetadataV3), // v0.15 - v????
```

Comment:
actually: this would hopefully correspond to `__data_version__ = 3`, rather then the API version `v0.15`, so in the end I think it is a good name :innocent: